test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -24,23 +24,26 @@ namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests
 {
     public class IntegrationTests
     {
-        private const string CollectorEndpointEnvVarName = "OTEL_EXPORTER_OTLP_ENDPOINT";
-        private static readonly string CollectorEndpoint = SkipUnlessEnvVarFoundFactAttribute.GetEnvironmentVariable(CollectorEndpointEnvVarName);
+        private const string CollectorEndpointEnvVarName = "OTEL_COLLECTOR_HOSTNAME";
+        private static readonly string CollectorEndpoint = SkipUnlessEnvVarFoundTheoryAttribute.GetEnvironmentVariable(CollectorEndpointEnvVarName);
 
+        [InlineData(OtlpExportProtocol.Grpc, 4317)]
+        [InlineData(OtlpExportProtocol.HttpProtobuf, 4318)]
         [Trait("CategoryName", "CollectorIntegrationTests")]
-        [SkipUnlessEnvVarFoundFact(CollectorEndpointEnvVarName)]
-        public void ExportResultIsSuccess()
+        [SkipUnlessEnvVarFoundTheory(CollectorEndpointEnvVarName)]
+        public void ExportResultIsSuccess(OtlpExportProtocol protocol, int port)
         {
 #if NETCOREAPP3_1
-            // Adding the OtlpExporter creates a GrpcChannel.
-            // This switch must be set before creating a GrpcChannel/HttpClient when calling an insecure gRPC service.
+            // Adding the OtlpExporter creates a GrpcChannel/HttpClient.
+            // This switch must be set before creating a GrpcChannel/HttpClient when calling an insecure HTTP/2 endpoint.
             // See: https://docs.microsoft.com/aspnet/core/grpc/troubleshoot#call-insecure-grpc-services-with-net-core-client
             AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
 #endif
 
             var exporterOptions = new OtlpExporterOptions
             {
-                Endpoint = new System.Uri($"http://{CollectorEndpoint}"),
+                Endpoint = new System.Uri($"http://{CollectorEndpoint}:{port}"),
```

Comment:
Not introduced by this PR, the term `CollectorEndpoint` seems to be confusing - perhaps endpoint should include the port?

Reply:
CollectorHostname will be better